internal/runners/packages/search.go

```diff
@@ -86,6 +80,56 @@ func targetedLanguage(languageOpt string) (string, *failures.Failure) {
 	return model.LanguageForCommit(proj.CommitUUID())
 }
 
+type modules []string
+
+func makeModules(normalizedName string, pack *model.IngredientAndVersion) modules {
```

Comment:
Wouldn't `newModules` be the idiomatic naming?

Reply:
Not in this case; The "new" and "make" prefix correlate to the "new" and "make" builtin functions. The make prefix is less common in general because return values tend to be pointers-to, but also because using the make prefix for non-pointer-to returns is not as widely known.

Reply:
Using the "make" prefix is practiced in the standard library, but not as consistently as "new": https://golang.org/src/cmd/internal/src/pos.go?s=1216:1306#L28